mne/viz/backends/renderer.py

```diff
@@ -158,11 +155,14 @@ def _get_3d_backend():
 
 @contextmanager
 def use_3d_backend(backend_name):
-    """Create a viz context.
+    """Create a 3d visualization context using the designated backend.
+
+    See :func:`mne.viz.set_3d_backend` for more details on the available
+    3d backends and their capabilities.
 
     Parameters
     ----------
-    backend_name : str
+    backend_name : 'mayavi' | 'pyvista' | 'notebook'
```

Comment:
I think the [numpydoc standard](https://numpydoc.readthedocs.io/en/latest/format.html) for this would be `backend_name : {'mayavi', 'pyvista', 'notebook'}` but locally we always just list these in the descriptions below.